template.json

```diff
@@ -921,12 +921,12 @@
                 },
                 "visibilityChecks": {
                     "astral": {
+                        "hasAura" : false,
                         "astralActive" : false,
                         "affectedBySpell": false
                     },
                     "meat": {
-                        "hidden": false,
-                        "hasHeat": true
+                        "hasHeat": false
```

Comment:
This will need a migration for existing actors of this type.
Foundry will recognize the meat.hasHeat: true and _not_ override that value. Foundry expects the values to be initial values.

Reply:
I wrote something. No clue if it works as intended

Reply:
Great. I'll check it out this weekend.

These types of migrations can easily be tested by placing some test actors on:
- the sidebar
- scene tokens
- compendium actors
- compendium scene tokens

As migrations can be annoying and the migration system is convoluted, I'll double check with you :)

Reply:
Was 90% correct. https://github.com/SR5-FoundryVTT/SR5-FoundryVTT/pull/895/commits/6863df60f753758a2e0fee64246193a104024a34
I've changed the the MigrateActorData to return only what needs to be updated instead of all actor data. This way Foundry is quicker on updating, as it can skip diffing before update. (Which it otherwise always does).
Besides that the v16 Migration needed to be activated into the list migrations.
I tested the migration and it looked good to me. :)